bedrock/firefox/templates/firefox/whatsnew/includes/fx101-vpn-mobile/qr-fr.html

```diff
@@ -0,0 +1,12 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="mzp-c-emphasis-box">
+  <h3 class="qr-title">Scannez le code QR dès maintenant</h3>
+  <div class="qr-code-wrapper">
+    <img src="{{ static('img/firefox/whatsnew/whatsnew101/qr-vpn-fr.svg') }}" width="400" height="400" alt="code QR">
```

Comment:
I'm guessing this is the same for all of the QR codes? Suggest checking / updating them all.